includes/framework/QDbBackedSessionHandler.class.php

```diff
@@ -29,13 +56,13 @@ class QDbBackedSessionHandler extends QBaseClass {
 		/**
 		 * @var string The session name to be used for saving sessions.
 		 */
-		protected static $strSessionName = '';
+		protected static $strSessionName	= ''; //PHPSESSID on a default PHP installation
 		
 		/** @var bool Whether to base64 the session data. Required when storing data in a TEXT field. */
-		public static $blnBase64 = true;
+		public static $blnBase64			= false;
```

Comment:
I am concerned about this. This is potentially a breaking change, and somewhat difficult to debug. Can you leave it as true, since that will work for everyone and false is really just a step that optimizes on databases that can handle straight binary?